style.css

```diff
@@ -1,5 +1,258 @@
-.nav-bar {
+/* HOME */
+
+.home {
+  background-image: url(./Pictures/Homepage.jpeg);
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+  background-position: center center;
+}
+
+.margin-t-l {
+  margin-top:100px;
+}
+
+
+.layout-container {
+  margin:20px 50px;
+}
+
+.home-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-family: "Oswald", sans-serif;
+  color: #202121;
+  /* margin: 0px 50px; */
+  height: 100vh;
+}
+
+.square {
+  background: rgba(1, 24, 40, 0.9);
+  border-radius: 5px;
+  font-weight: bold;
+  color: #f8f8f8;
+  font-size: 3rem;
+  text-align: center;
+  padding:0 30px;
+  
+}
+
+.button {
+  color: #f8f8f8;
+  border: solid 1px #f8f8f8;
+  border-radius: 5px;
+  text-decoration: none;
+  font-size: 1rem;
+  padding: 5px;
+  margin: 20px 100px;
+  display: flex;
+  justify-content: center;
+}
+
+.button:hover {
```

Comment:
Any values that you gave to the button class will be inherited on the hover state, so you can remove them.